src/lib/room.js

```diff
@@ -185,11 +185,13 @@ Livechat.onMessage(async (message) => {
 });
 
 export const getGreetingMessages = (messages) => messages && messages.filter((msg) => msg.trigger);
+export const getLatestCallMessage = (messages) => messages && messages.filter((msg) => msg.actionLinks && msg.actionLinks.length === 2);
```

Comment:
I didn't understand why we're adding such vague 
```suggestion
export const getLatestCallMessage = (messages) => messages && messages.filter((msg) => (msg.t === 'livechat_webrtc_video_call' || msg.t === 'livechat_video_call') || !msg.endTs).pop();
```
To me this logic makes more sense because we're checking the message type instead of the action links array length. Remember there can be some cases where people would be adding two buttons/actionLinks to their livechat message, and if we keep your logic here - it would treat such messages too as call messages and the logic applied would fail.